app/src/main/java/com/github/orkest/View/MainActivity2.kt

```diff
@@ -0,0 +1,72 @@
+package com.github.orkest.View
+
+
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import com.github.orkest.R
+import com.spotify.android.appremote.api.ConnectionParams
+import com.spotify.android.appremote.api.Connector.ConnectionListener
+import com.spotify.android.appremote.api.SpotifyAppRemote
+import com.spotify.protocol.types.PlayerState
+
+/**
+ * Activity used to test the Spotify API (DEMO ONLY)
+ */
+class MainActivity2 : AppCompatActivity() {
+    private var mSpotifyAppRemote: SpotifyAppRemote? = null
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main2)
+    }
```

Comment:
Have you chosen to use xml or are you going back to JetPack Compose for this activity ? (curious)

Reply:
That was just for testing so he did it as quick as possible